### PR TITLE
Split Debugger::outputAs() into getter/setter

### PR DESCRIPTION
**Pull request by cleptric**
Continues with #9978

### CODE REVIEW

File: src/Error/Debugger.php [hunk 1]
```diff
@@ -651,8 +651,40 @@ protected static function _object($var, $depth, $indent)
...
+        $self = Debugger::getInstance();
+
+        return $self->_outputFormat;
+    }
+
+    /**
+     * Set the output format for Debugger error rendering.
+     *
+     * @param string $format The format you want errors to be output as.
+     * @return null
```
Comment (dereuromark) [inline src/Error/Debugger.php]: should be void :)

File: src/Error/Debugger.php [hunk 2]
```diff
@@ -651,8 +651,40 @@ protected static function _object($var, $depth, $indent)
...
+    public static function setOutputAs($format)
+    {
+        $self = Debugger::getInstance();
+
+        if (!isset($self->_templates[$format])) {
+            throw new InvalidArgumentException('Invalid Debugger output format.');
+        }
+        $self->_outputFormat = $format;
+
+        return null;
```
Comment (dereuromark) [inline src/Error/Debugger.php]: this is not needed then.

File: src/Error/Debugger.php [hunk 3]
```diff
@@ -651,8 +651,38 @@ protected static function _object($var, $depth, $indent)
...
     /**
+     * Get the output format for Debugger error rendering.
+     *
+     * @return string Returns the current format when getting.
+     */
+    public static function getOutputAs()
+    {
+        $self = Debugger::getInstance();
+
+        return $self->_outputFormat;
```
Comment (markstory) [inline src/Error/Debugger.php]: Couldn't this be a one liner?

Comment (ravage84) [inline src/Error/Debugger.php]: Why not ``getOutputFormat()``?

> Get the **output format** for Debugger error rendering.

Comment (cleptric) [inline src/Error/Debugger.php]: I'll do a followup. It should be also be `setOutputFormat()` then

**Pull request merged**